syntax = "proto3";

package opentera.protobuf;

message StopSessionEvent {
    string session_uuid = 1;
    string service_uuid = 2;
}
